### ISSUE TITLE
Rectify11 breaks reWASD

### ISSUE THREAD

**Issue by Nathan22211** [Rectify11/Installer#504]
### Description

when this software is installed with reWASD, reWASD doesn't work and gives an error code of -1

### Steps To Reproduce

install recify with reWASD, the latter should fail to startup on reboot

### Rectify11 Version

3.1

### Windows Version

22H2 don't remember the excact build number but it was the latest one at the time of posting

### Logs

this is related to a conflict with another software and I don't have any logs to go off of as to what broke in reWASD, save for the error code

**Comment by Usernames-father** [comment]
Yeah, I have the same issue, it seems like no solution is coming. It is either rectify or rewasd to choose!( Tried different versions but no luck.
Rectify11 Version
3.0-3.2
ReWasd Version
7.0-7.3
Windows 11 Version
23H2

**Comment by Vehlore** [comment]
@MishaProductions I have found a workaround for this issue that may be good to put in the FAQ together with the known error.

After long investigating on DLLs (I didn't have any idea on what each DLL modified lol) I came up with a list of files that might affect reWASD functionality. I made a Rectify11 reinstall unselecting some DLLs and reWASD ended up working fine with 99% of the Rectify11 tweaks working as well.

Here's the list:

**input.dll.mui** - most likely the cause since this messes with DirectInput and XInput.

**usbui.dll** - might affect wired controllers.
**bthprops.cpl** - might affect BT controllers.
**bthci.dll** - might affect BT controllers.

**devmgr.dll** - unlikely to affect but skipped it just in case
**joy.cpl** - unlikely to affect but skipped it just in case
**hdwwiz.cpl** - unlikely to affect but skipped it just in case

Probably somebody would be able to narrow it down more than I did because I don't have a lot of time to test all of those files one by one but I hope this helps in some way. At least users can have Rectify11 isntalled while still using reWASD.
